stockfish/models.py

```diff
@@ -283,6 +297,38 @@ def is_move_correct(self, move_value: str) -> bool:
                 else:
                     return True
 
+    def get_WDL_stats(self) -> List:
+        """Returns Stockfish's win/draw/loss stats for the side to move.
+
+        Returns:
+            Ideally, a list of three integers. However, if the game is over,
+            or if too old a version of SF is being used, then an empty list
+            is returned.
+        """
+
+        if "UCI_ShowWDL" not in self._parameters:
+            return []
+        self._go()
+        lines = []
+        while True:
+            text = self._read_line()
+            splitted_text = text.split(" ")
+            lines.append(splitted_text)
+            if splitted_text[0] == "bestmove":
+                break
+        for current_line in reversed(lines):
+            if current_line[0] == "bestmove" and current_line[1] == "(none)":
+                return []
+            elif "multipv" in current_line:
+                index_of_multipv = current_line.index("multipv")
+                if current_line[index_of_multipv + 1] == "1":
+                    index_of_wdl = current_line.index("wdl")
+                    wdl_stats = []
+                    for i in range(1, 4):
+                        wdl_stats.append(int(current_line[index_of_wdl + i]))
+                    return wdl_stats
+        raise RuntimeError("Reached the end of the get_WDL_stats function.")
```

Comment:
Let's create test case to catch this `RuntimeError`

Reply:
Thanks for your feedback, I've implemented and pushed most of the changes. 
For this RuntimeError, the code shouldn't reach it (even if the user is using an older version of SF without WDL output). So in order to make it be thrown, either there would have to be a bug in this function, or maybe the user tries doing something like multithreading. 
I also changed some of the WDL function names to be clearer for what they do. The main thing is trying to indicate that the WDL option can be enabled/disabled (for more recent SFs) or not a UCI option at all (SF 11 and older).